openedx_events/content_authoring/signals.py

```diff
@@ -210,3 +211,16 @@
         "content_object": ContentObjectData
     }
 )
+
+# .. event_type: org.openedx.content_authoring.xblock.lti1p1.content.graded.v1
+# .. event_name: XBLOCK_LTI1P1_GRADED
+# .. event_key_field: xblock.scope_ids.usage_id
+# .. event_description: emitted when an LTI 1.1 Content is graded via Xblock's OutcomeService
+# .. event_data: Lti1p1ContentGraded
+# .. event_trigger_repository: Pearson-Advance/xblock-lti-consumer
+XBLOCK_LTI1P1_GRADED = OpenEdxPublicSignal(
+    event_type="org.openedx.content_authoring.xblock.lti1p1.content.graded.v1",
+    data={
+        "graded_content": Lti1p1ContentGraded
+    }
```

Comment:
Trailing comma. @Nekenhei

Reply:
We should not follow bad practices from others, we should follow the best practices, even if the rest of the file has bad practices, you should not use that as an argument to continue having coding with bad practices. @Nekenhei